tests/testthat/test-object_usage_linter.R

```diff
@@ -480,3 +480,15 @@ test_that("unknown infix operators give good lint metadata", {
     object_usage_linter()
   )
 })
+
+test_that("respects `skip_with` argument for `with()` expressions", {
+  f <- tempfile()
+  writeLines(
+    "test_fun <- function(df) {
+      with(df, first_var + second_var)
+     }",
+    f
+  )
+  expect_true(length(lint(f, object_usage_linter(skip_with = TRUE))) == 0L)
```

Comment:
use `expect_length()` here

Reply:
Done.

Should we be including the `expect_*()` linters in lint changed files GHA?

Reply:
I was wondering the same... works for me

Reply:
Done: https://github.com/r-lib/lintr/pull/1557